### ISSUE TITLE
Wildcard V2 Feedback: Figma and codebase alignment

### ISSUE THREAD

**Issue by taylorsperry** [sourcegraph/sourcegraph#28992]

**Comment by oleggromov** [comment]
I agree that having variants, colors, and other _labels_ synchronized is a great and important idea - what I am wondering, however, is how automated we could/should make this? I.e. how often do design files get updated?
